pkg/client/keycloak/config.go

```diff
@@ -95,6 +100,12 @@ func NewKeycloakConfig() *KeycloakConfig {
 			GrantType:        "client_credentials",
 			Scope:            "api.iam.service_accounts",
 		},
+		InternalSSORealm: &KeycloakRealmConfig{
+			BaseURL:        "https://auth.redhat.com",
+			APIEndpointURI: "/auth/realms/EmployeeIDP",
+			Realm:          "EmployeeIDP",
+		},
+		InternalSsoBaseURL:                         "https://auth.redhat.com",
```

Comment:
Probably this won't be needed if https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1293/files#r979956339 is addressed